aws/data_source_awslogsdelivery_canonical_user_id_test.go

```diff
@@ -0,0 +1,44 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"testing"
+)
+
+func TestAwslogsdeliveryCanonicalUserId(t *testing.T) {
```

Comment:
[ask] what point is lacked for acceptant test pass?

This test output is like this now.

```
        Error: Invalid data source
        
          on ../../../../../../../../var/folders/lg/rdr0tvnd6kzblb0y1xmpvvx00000gn/T/tftest859718032/work866635183/config898707539/terraform_plugin_test.tf line 2, in data "aws_awslogsdelivery_canonical_user_id" "main":
           2: data "aws_awslogsdelivery_canonical_user_id" "main" {}
        
        The provider provider.aws does not support data source
        "aws_awslogsdelivery_canonical_user_id".
```

I added `aws_awslogsdelivery_canonical_user_id` to aws/provider.go, but failed. Should I need to modify any other codes to register new data source name?

Reply:
I applied 55418e6106dc93b3bd4e2ab7faa7027ba92169d7 to your branch to appease the `awsproviderlint` failures, but also re-executed the acceptance tests and they completed successfully:

```console
$ AWS_PROFILE="hector" make testacc TESTARGS='-run=TestAccAwslogsdeliveryCanonicalUserId_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwslogsdeliveryCanonicalUserId_basic -timeout 120m
=== RUN   TestAccAwslogsdeliveryCanonicalUserId_basic
=== PAUSE TestAccAwslogsdeliveryCanonicalUserId_basic
=== CONT  TestAccAwslogsdeliveryCanonicalUserId_basic
--- PASS: TestAccAwslogsdeliveryCanonicalUserId_basic (7.40s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       7.425s
```

Reply:
I'm sorry to respond late. Thank you, I'll try it again.

Reply:
passed! thanks!

```
AWS_PROFILE="hgsgtk" make testacc TESTARGS='-run=TestAccAwslogsdeliveryCanonicalUserId_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwslogsdeliveryCanonicalUserId_basic -timeout 120m
=== RUN   TestAccAwslogsdeliveryCanonicalUserId_basic
=== PAUSE TestAccAwslogsdeliveryCanonicalUserId_basic
=== CONT  TestAccAwslogsdeliveryCanonicalUserId_basic
--- PASS: TestAccAwslogsdeliveryCanonicalUserId_basic (37.91s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	39.560s
```